config.yaml

```diff
@@ -38,5 +38,13 @@ metadata:
     enable: false
 
 userscripts:
-    setup: ./setup_cice_restarts.sh
-    archive: ./archive_cice_restarts.sh
+    setup: if [ -f work/access-om3.cice.r.* ] ; then RESTART=$(echo work/access-om3.cice.r.*) ; ln -f $(readlink $RESTART) $RESTART ; fi
```

Comment:
Sorry, I forgot that the setup script is obviously also needed for continue runs.

I think I've changed my mind - could we modify `om3-scripts/payu_config/setup_scripts/setup_cice_restarts.sh` so that it only replaces the `iced.1900-01-01-10800.nc` symlink if it exists.

Then this part of the `config.yaml` can be the same across configs and so automatic cherry-picking, e.g. to remove the `setup` `userscript`, will be more likely to work.

Reply:
fixled :)